source/localizable/templates/some-basic-helpers.md

```diff
@@ -0,0 +1,42 @@
+## Some Basic Built in Helpers
+
+In the last section you learned how to write a helper.
```

Comment:
I just read the previous section, and it feels weird to both have it before this section, and then have this section, and then go back to writing helpers.
Can you move the "Helpers" section from that file to the end of this one? Connect both of them by saying something like "Besides using the built-in helpers, you can also write your own"